src/components/useTracker.js

```diff
@@ -1,33 +1,44 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useRef, useReducer, useMemo } from 'react';
 import Tracker from '../Tracker.js';
-import Data from '../Data';
 
+const fur = (x: number): number => x + 1;
+const useForceUpdate = () => useReducer(fur, 0)[1];
 export default (trackerFn, deps = []) => {
```

Comment:
Please add some documentation for this module, especially how it behaves, preferrably jsDoc; optimally would be to add comments on each of the React hooks so other devs understand how they work and why they were chosen

Reply:
This file was copied from meteor-react-data at some point, I don't really have an amazing understanding enough to do anything with this.